sbi/inference/potentials/score_based_potential.py

```diff
@@ -192,8 +213,8 @@ def get_continuous_normalizing_flow(
 def build_freeform_jacobian_transform(
     score_estimator: ConditionalScoreEstimator,
     x_o: Tensor,
-    atol: float = 1e-5,
-    rtol: float = 1e-6,
+    atol: float = 1e-6,
+    rtol: float = 1e-5,
```

Comment:
was this a bug or just updates because these values work better empirically?

Reply:
I think this was a bug, as the default values for `FreeFormJacobianTransform` in zuko are the new values after the change. I am guessing this was a typo when we initially created this function. It makes sense for `atol`<`rtol`.